src/SampleWebApp/Generated/Nox.Generator/Nox.Generator.NoxCodeGenerator/SampleWebAppDbContext.g.cs

```diff
@@ -42,20 +42,34 @@ protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
         }
     }
     
+    public static Type? GetTypeByEntityName(string entityName)
+    {
+        return Type.GetType("SampleWebApp.Domain." + entityName);
+    }
+    
     protected override void OnModelCreating(ModelBuilder modelBuilder)
     {
         base.OnModelCreating(modelBuilder);
         if (_noxSolution.Domain != null)
         {
             var codeGeneratorState = new NoxSolutionCodeGeneratorState(_noxSolution);
             
+            var relationships = ((INoxDatabaseConfigurator)_dbProvider).GetRelationshipsToCreate(codeGeneratorState, _noxSolution.Domain.Entities, modelBuilder);
```

Comment:
The relations to create is a Nox.Solution concept not a databaconfigurator, the Configurator should only configure...